### DISCUSSION TITLE
Add total size of the selected files/directories somewhere in the status bar?

### DISCUSSION THREAD

**Discussion by jpggithub** [gokcehan/lf#1521] [General]
I would like to always display the total size of the selected files/directories somewhere in the status bar.

Inspired by https://github.com/gokcehan/lf/discussions/1214#discussioncomment-5631498, I tried:
```
cmd on-select %du -ch $fx | tail -n 1 | cut -f 1
```
 but it deletes the information displayed at the bottom left each time I move the selection (because a new file/directory is selected by default at each move).

Is there any way of displayed the total size of selected files/directories without deleting something? 
It could be an option for the ruler ? (for example next `df`)

**Comment by joelim-work** [comment]
You can use [user-defined options](https://github.com/gokcehan/lf/blob/master/doc.md#user_option-string-default-none) to store custom data and display it in the ruler:

```shell
cmd on-select &{{
    result="$(du -ch $fx | tail -n 1 | cut -f 1)"
    lf -remote "send $id set user_du $result"
}}

set rulerfmt '  %{lf_user_du}|  <everything_else>'
```

**Comment by jpggithub** [comment]
Thank you (even if I have to wait the r31 version in Debian Testing to see the `rulerfmt` in the manual ;-)
